app/src/test/java/io/hstream/testing/TestUtils.java

```diff
@@ -306,6 +306,7 @@ public static Consumer createConsumerCollectStringPayload(
   }
 
   public static void restartServer(GenericContainer<?> server) throws Exception {
+    Thread.sleep(1000);
```

Comment:
Is this necessary?

Reply:
https://github.com/hstreamdb/integration-tests/pull/27
and somewhere had case when from log file server is started, but refused to connect

Reply:
Ok, I got it.